src/Axiom/Set.agda

```diff
@@ -84,6 +89,20 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   weakly-finite : Set A → Type ℓ
   weakly-finite X = ∃[ l ] ∀ {a} → a ∈ X → a ∈ˡ l
 
+  strongly-finite : Set A → Type ℓ
+  strongly-finite X = ∃[ l ] Unique l × ∀ {a} → a ∈ X ⇔ a ∈ˡ l
```

Comment:
Is this the first place `Unique` is used in the formal-ledger-spec?  If so, should we add a comment or doc explaining what it is?  e.g., 

```agda
  -- `Unique l` is an alias for `Allpairs R l`, where the relation R is
  -- `Distinct x y = ¬ (x ≈ y)`, so `Unique l` means `x ≉ y` for all `x, y ∈ l`.
```

Reply:
That's a fair point, I first read it as "exists a unique `l`"

Reply:
I'll add a comment explaining that definition a bit and then merge this PR. I'm happy to rework the definition of `card` in some other way in the future.